cmd/bootstrap/cmd/constraints.go

```diff
@@ -12,6 +12,9 @@ func ensureUniformNodeWeightsPerRole(allNodes flow.IdentityList) {
 	// ensure all nodes of the same role have equal weight
 	for _, role := range flow.Roles() {
 		withRole := allNodes.Filter(filter.HasRole(role))
+		if len(withRole) == 0 {
+			continue
+		}
```

Comment:
```suggestion
```

I'm sure this is already checked elsewhere, but having 0 nodes for any role is an invalid configuration. Better to let the bootstrapping panic (fail fast) than silently continue with a known invalid input.